src/elements/inputGroup/InputGroup.types.ts

```diff
@@ -1,9 +1,19 @@
 interface Props {
+  value: string;
+  id?: string;
+  className?: string;
+  inputLabelProps?: any;
```

Comment:
This "inputLabelProps" should have its type imported from the interface of the "InputLabel" component. Can you create a file name "InputLabel.types.ts" with its interface in the component folder, and import that interface in every inputLabelProps prop across the toolkit?

Reply:
Done 👍